pallets/ddc-customers/src/mock.rs

```diff
@@ -0,0 +1,154 @@
+//! Test utilities
+
+#![allow(dead_code)]
```

Comment:
don't use it globally but for specific line - even in mock/tests

Reply:
+